install/kubernetes/experimental-install.yaml

```diff
@@ -351,6 +351,14 @@ rules:
   - get
   - list
   - watch
+- apiGroups:
+  - coordination.k8s.io
+  resources:
+  # To be used during HA deployment of cilium-operator in order
+  # to obtain lease lock for leader election.
+  - leases
+  verbs:
+  - '*'
```

Comment:
Will we need the same check as above on the K8s version?

Reply:
These are autogenerated files using the helm templates. I don't think we can add checks to these files, as they will be overridden next time we autogenerate install manifests.